tests/functionnal/ReminderTranslation.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2020 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+*/
+
+namespace tests\units;
+
+use \DbTestCase;
+
+/* Test for inc/ReminderTranslation.class.php */
+
+/**
+ * @engine isolate
+ */
+class ReminderTranslation extends DbTestCase {
+
+   public function testGetTranslationForReminder() {
+
+      $this->login();
+      $this->setEntity('_test_root_entity', true);
+
+      $date = date('Y-m-d H:i:s');
+      $_SESSION['glpi_currenttime'] = $date;
+
+      $data = [
+         'name'         => '_test_reminder01',
+         'entities_id'  => 0
+      ];
+
+      $reminder = new \Reminder();
+      $added = $reminder->add($data);
+      $this->integer((int)$added)->isGreaterThan(0);
+
+      $reminder1 = getItemByTypeName(\Reminder::getType(), '_test_reminder01');
+
+      //first, set data
+      $text_orig = 'Translation 1 for Reminder1';
+      $text_fr = 'Traduction 1 pour Note1';
+      $this->addTranslation($reminder1, $text_orig);
+      $this->addTranslation($reminder1, $text_fr, 'fr_FR');
+
+      $nb = countElementsInTable(
+         'glpi_remindertranslations'
+      );
+      $this->integer((int)$nb)->isIdenticalTo(2);
+
+      // second, test what we retrieve
+      $_SESSION['glpilanguage'] = 'fr_FR';
+      $text = \ReminderTranslation::getTranslatedValue($reminder1, "text");
+      $this->string($text)->isIdenticalTo($text_fr);
```

Comment:
```suggestion
      $current_lang = $_SESSION['glpilanguage'];
      $_SESSION['glpilanguage'] = 'fr_FR';
      $text = \ReminderTranslation::getTranslatedValue($reminder1, "text");
      $_SESSION['glpilanguage'] = $cur_lang;
      $this->string($text)->isIdenticalTo($text_fr);
```

Not sure this is required, but resetting as it was is IMHO never a bad idea.